### DISCUSSION TITLE
Follow path script

### DISCUSSION THREAD

**Discussion by JR2012thman** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#709] [Q&A]
Instead of reinventing the wheel I found in the documentation that there is a follow path option and you have to set up the script for it. ok seems easy enough, but that functionality has changed or I don't understand it well so can anyone help me? I tried pasting in the yaml into the frontend script creator for HA but no joy. Do I need to manually add this to the scripts file?

**Comment by PiotrMachowski** [comment]
To add it in UI you have to skip the first line. 

Generally speaking this script isn't really useful: it is made by sending the vacuum to the point, waiting for it to reach it, then sending to a next point. Vacuums usually do not clean when going to the target.
